code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -213,7 +213,7 @@
 		if("Miner")
 			module = new /obj/item/weapon/robot_module/miner(src)
 			hands.icon_state = "miner"
-			icon_state = "minerborg"
+			icon_state = "ashborg"
```

Comment:
The same reason we don't let miners chose between lavaland-styled gear and old gear.